gulpfile.js

```diff
@@ -0,0 +1,43 @@
+// enable debug for gulp
+process.env.DEBUG = process.env.DEBUG || 'fcc:*';
+require('dotenv').load();
+
+require('babel-core/register');
+
+const gulp = require('gulp'),
+  debug = require('debug')('fcc:gulp'),
+  path = require('path'),
+  nodemon = require('gulp-nodemon');
+
+gulp.task('serve', function(cb) {
+  let called = false;
+  const monitor = nodemon({
+    script: path.normalize('node_modules/serverless/bin/serverless'),
```

Comment:
I'm just wondering.. what is the best way to do this?

If people have installed using `yarn global add serverless` this could be installed in `$(yarn global dir)/node_modules/.bin`.

Should we test if they have serverless in their PATH first, and if not test first `$(yarn global dir)/node_modules/.bin/serverless` exists, and next `node_modules/serverless/bin/serverless`?

I'm fine with merging this as is, just something I was wondering in general, can be fixed in another PR too.

Reply:
Well that would be ideal, I am not sure if the version they are on would be at par that we use? This at least ensures that the one on the package.json (which we are using most probably) is being used.

Another option is to use https://github.com/zkat/npx which comes packaged with npm but that is the path which we have left with moving over to yarn

Reply:
I think declaring a version for people to use is the best way. Serverless could have a major version bump and things could start breaking unexpectedly for contributors.